docs/source/aws/gettingstarted.rst

```diff
@@ -60,3 +78,34 @@ For more information on basic concepts and terms, check the :ref:`glossary
 <glossary>`. See our extended example of a policy's structure 
 :ref:`tag compliance policy <policyStructure>`, or browse all of our
 :ref:`use case recipes <usecases>`.
+
+.. _monitor-aws-cc:
+
+Monitor AWS
+-----------
+
```

Comment:
missing content on using xray tracing

Reply:
although perhaps that goes into an  ops section, or advanced aws

Reply:
yeah i think it's best to push that later down the line